src/patternfly/components/Card/card-header.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="pf-c-card__header pf-c-title pf-m-lg{{#if card-header--modifier}} {{card-header--modifier}}{{/if}}"
+<div class="pf-c-card__header pf-c-title pf-m-md{{#if card-header--modifier}} {{card-header--modifier}}{{/if}}"
```

Comment:
Has @mceledonia looked at this, too? This just changed recently as a result of this conversation with the Integreatly team - https://github.com/patternfly/patternfly-next/issues/626#issuecomment-466500428